bigtable/hello/main.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Demonstrates how to connect to Cloud Bigtable and run some basic operations.
+
+Prerequisites:
+
+- Create a Cloud Bigtable cluster.
+  https://cloud.google.com/bigtable/docs/creating-cluster
+- Set your Google Application Default Credentials.
+  https://developers.google.com/identity/protocols/application-default-credentials
+- Set the GCLOUD_PROJECT environment variable to your project ID.
+  https://support.google.com/cloud/answer/6158840
+"""
+
+import argparse
+
+from gcloud import bigtable
+
+
+def main(project_id, cluster_id, zone, table_id):
+    # [START connecting_to_bigtable]
+    # The client must be created with admin=True because it will create a
+    # table.
+    client = bigtable.Client(project=project_id, admin=True)
+    with client:
+        cluster = client.cluster(zone, cluster_id)
+        # [END connecting_to_bigtable]
+
+        # [START creating_a_table]
+        print('Creating the {} table.'.format(table_id))
+        table = cluster.table(table_id)
+        table.create()
+        column_family_id = 'cf1'
+        cf1 = table.column_family(column_family_id)
+        cf1.create()
+        # [END creating_a_table]
+
+        # [START writing_rows]
+        print('Writing some greetings to the table.')
+        column_id = 'greeting'.encode('UTF-8')
+        greetings = [
+            'Hello World!',
+            'Hello Cloud Bigtable!',
+            'Hello Python!',
+        ]
```

Comment:
/shruggie

It's more a personal nit of mine than something that's been codified.